### PR TITLE
COMMERCE-5721 Fix minium login page's web content

### PR DESCRIPTION
**Pull request by alecsloan**

### CODE REVIEW

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

**Review by dsitu — APPROVED** [review]

File: modules/apps/commerce/commerce-theme-minium/commerce-theme-minium-site-initializer/src/main/resources/com/liferay/commerce/theme/minium/site/initializer/internal/dependencies/journal_articles/LOGIN_SPLASH.xml [hunk 1]
```diff
@@ -1,12 +1,9 @@
...
-<root available-locales="[$LOCALE$]" default-locale="[$LOCALE$]">
+<root available-locales="[$LOCALE$]" default-locale="[$LOCALE$]" version="1.0">
 	<dynamic-element index-type="text" instance-id="mewm" name="splashImage" type="image">
 		<dynamic-content language-id="[$LOCALE$]">
-		<![CDATA[[$login_portlet_splash.jpg$]]]>
+			<![CDATA[[$login_portlet_splash.jpg$]]]>
 		</dynamic-content>
-		<dynamic-element index-type="keyword" instance-id="mewm" name="alt" type="text">
-			<dynamic-content language-id="[$LOCALE$]"><![CDATA[Minium Logo]]></dynamic-content>
-		</dynamic-element>
```
Comment (ivicac) [inline modules/apps/commerce/commerce-theme-minium/commerce-theme-minium-site-initializer/src/main/resources/com/liferay/commerce/theme/minium/site/initializer/internal/dependencies/journal_articles/LOGIN_SPLASH.xml:10]: @alecsloan these elements are not used anymore?

File: modules/dxp/apps/commerce-demo-pack/commerce-theme-minium/commerce-theme-minium-full-site-initializer/src/main/resources/com/liferay/commerce/theme/minium/full/site/initializer/internal/dependencies/journal_articles/LOGIN_SPLASH.xml [hunk 2]
```diff
@@ -1,12 +1,9 @@
...
-<root available-locales="[$LOCALE$]" default-locale="[$LOCALE$]">
+<root available-locales="[$LOCALE$]" default-locale="[$LOCALE$]" version="1.0">
 	<dynamic-element index-type="text" instance-id="mewm" name="splashImage" type="image">
 		<dynamic-content language-id="[$LOCALE$]">
-		<![CDATA[[$login_portlet_splash.jpg$]]]>
+			<![CDATA[[$login_portlet_splash.jpg$]]]>
 		</dynamic-content>
-		<dynamic-element index-type="keyword" instance-id="mewm" name="alt" type="text">
-			<dynamic-content language-id="[$LOCALE$]"><![CDATA[Minium Logo]]></dynamic-content>
-		</dynamic-element>
```
Comment (ivicac) [inline modules/dxp/apps/commerce-demo-pack/commerce-theme-minium/commerce-theme-minium-full-site-initializer/src/main/resources/com/liferay/commerce/theme/minium/full/site/initializer/internal/dependencies/journal_articles/LOGIN_SPLASH.xml:10]: @alecsloan these elements are not used anymore

**Review by ivicac — CHANGES_REQUESTED** [review]

**Review by ivicac — APPROVED** [review]

**Comment by ivicac** [comment]
ci:forward

**Pull request closed without merging**